osu.Game/Rulesets/UI/FrameStabilityContainer.cs

```diff
@@ -191,7 +193,9 @@ private void setClock()
 
         private class StabilityGameplayClock : GameplayClock
         {
-            public IFrameBasedClock ParentGameplayClock;
+            public GameplayClock ParentGameplayClock;
+
+            public override IEnumerable<Bindable<double>> NonGameplayAdjustments => ParentGameplayClock.NonGameplayAdjustments;
```

Comment:
`ParentGameplayClock` could be null, if the parent clock wasn't a `GameplayClock`.

```suggestion
            public override IEnumerable<Bindable<double>> NonGameplayAdjustments => ParentGameplayClock?.NonGameplayAdjustments;
```

Reply:
How come this doesn't fail any tests?

Reply:
Probably because there isn't any code path to this property in all existing tests, I still think adding a null conditional would be better to avoid a potential null reference.